src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp

```diff
@@ -1488,6 +1501,10 @@ namespace OpenMS
   {
     if (!ws_.currentSubWindow())
     {
+      if (activeSubwindowID_ < (Size) ws_.subWindowList().size())
```

Comment:
i'm not sure what this is doing...
if the subwindow is currently empty, why return the "old" one?
What happens if the window was closed?

Reply:
I think it had something to do with focus.
If the plot area is not focussed, you get a nullptr for currentSubwindow.
But I think some other code depends on getting information from the currently visible (but not active)
spectra plot. 
Not sure if I explained that well.